### ISSUE TITLE
[GSSoC'21] Index Page

### ISSUE THREAD

**Issue by avinashjha1320** [ChetnaNSharma/Alumni-Connect#3]
@mksharma1 @yash982000 @ChetnaNSharma 
I would like to make the index page using HTML CSS and javascript with awesome animations and user experience.
Can you assign this to me...I want to build the site from scratch.

**Comment by yash982000** [comment]
@avinashjha1320 We are currently designing the screens. Will get back to you soon.
CC:- @ChetnaNSharma

**Comment by yash982000** [comment]
@avinashjha1320 I have uploaded the design screens in the folder.
You have to make it similar in HTML, CSS, and JavaScript.
Making it responsive will be a great thing and we will mark your contribution with proper points and label if you are able to do it nicely.

**Comment by tripathi-abhishek** [comment]
Assign me, please. I would like to work on it.

**Comment by yash982000** [comment]
> Assign me, please. I would like to work on it.

@tripathi-abhishek You will get different tasks. Head over to open to ideas issue listed and bring in some creative ideas you want to work on. Meanwhile, we are preparing the exact requirements and featured to be needed for this application. You can come to us with your ideas in the issue thread of Open for Ideas.
CC:- @ChetnaNSharma

**Comment by yash982000** [comment]
@tripathi-abhishek I am assigning you these screens if you can do it in a couple of next days let me know so that I can assign it to you.
https://github.com/ChetnaNSharma/Alumni-Connect/blob/main/Alumni%20Connect%20Design%20Screens/Alumni%20Connect%20Home-Feed%20Page%20-%201.png

https://github.com/ChetnaNSharma/Alumni-Connect/blob/main/Alumni%20Connect%20Design%20Screens/Alumni%20Connect%20Home-Feed%20Page%20-%202.png

**Comment by tripathi-abhishek** [comment]
ok i will do my best!! Please assign.

**Comment by yash982000** [comment]
> ok i will do my best!! Please assign.

Done!!

**Comment by Shagufta08** [comment]
Hey! @yash982000 , I would like to contribute to this project, Please assign me any page if it's not been assigned to anyone.

**Comment by yash982000** [comment]
> ok i will do my best!! Please assign.

@tripathi-abhishek Any updates??

**Comment by yash982000** [comment]
> Hey! @yash982000 , I would like to contribute to this project, Please assign me any page if it's not been assigned to anyone.

@Shagufta08 There are some designs to be done yet. Please open a new issue and I will assign you there.

**Comment by tripathi-abhishek** [comment]
> > ok I will do my best!! Please assign.
> 
> @tripathi-abhishek Any updates??

yeah. I'm building the pages as requested. The design is kinda vague. I have worked on Figma so I had this habit of getting precise details.  BTW I'm using HTML, CSS for now. So, In the future, if u need ejs, gatsby, or such for full website development it would just need some additional modules/components.

**Comment by yash982000** [comment]
> > > ok I will do my best!! Please assign.
> > 
> > 
> > @tripathi-abhishek Any updates??
> 
> yeah. I'm building the pages as requested. The design is kinda vague. I have worked on Figma so I had this habit of getting precise details. BTW I'm using HTML, CSS for now. So, In the future, if u need ejs, gatsby, or such for full website development it would just need some additional modules/components.

@tripathi-abhishek  They are actually not vague. It is made in Figma only. But, this is a screenshot so. No worries, do it in HTML and CSS only and connect it via Javascript if needed.

**Comment by samgarg7009** [comment]
hey! @yash982000, I would like to contribute to this project ,please assign me the designing task ,if not assigned to anyone

**Comment by yash982000** [comment]
@samgarg7009 What designs are you talking about?
We do have that ready.

**Comment by CMELAKKIA** [comment]
@yash982000 If would like to do the frontend work .If you have any designs ready can you assign me?

**Comment by Mukta-Sawant** [comment]
@yash982000 I would like to work on this issue as part of GSSOC'21. Please assign it to me,

**Comment by yash982000** [comment]
@CMELAKKIA @Mukta-Sawant There are certain designs in the design folder. You can check from the open and closed Pr's and issues and work on those which are left.

**Comment by ShrutiKhatri826** [comment]
I would like to contribute to this issue as a GSSOC'21 participant. Please assign it to me, hoping for a positive reply soon.

**Comment by yash982000** [comment]
> I would like to contribute to this issue as a GSSOC'21 participant. Please assign it to me, hoping for a positive reply soon.

@ShrutiKhatri826 Can you open a new issue?

You can contribute with your ideas or look for the remaining work from the repo

:)
